.travis.yml

```diff
@@ -13,26 +12,34 @@
 # limitations under the License.
 
 sudo: required
-
-language: python
-
-python:
-  - "3.6"
-
-services:
-  - docker
-
-before_install:
-  - ./install/scripts/docker_setup_forseti.sh
-
-install:
-  -
-
-script:
-  - ./install/scripts/docker_run_forseti.sh
-  - ./install/scripts/docker_unittest_forseti.sh
-  - ./install/scripts/docker_pystyle_forseti.sh
-
-after_success:
-  # Collect codecov results.
-  - docker -l error exec -it build /bin/bash -c "bash <(curl -s https://codecov.io/bash)"
+jobs:
+  include:
+  - stage: Integration test
```

Comment:
Can you please explain having different jobs here?  Will the Integration test run when a PR is created or updated?

Reply:
Updated `.travis.yml`. PTAL. Integration tests will not run for every PR. Newly added check was not included before for testing of this PR on Travis.